ArthurAtangana/club-management-system
# Refactor registration usecase with new schema changes

## What type of PR is this? (check all applicable)


## Description

Update registration use case to follow recent schema changes.

## How to test

run the cli, register a new member and check the table.
run it again and attempt to register the member again, it should fail and bring you back to the home screen.

## Added/updated tests?


## Screenshots
![refactorregistration](https://github.com/ArthurAtangana/club-management-system/assets/83664112/696063c1-c4d3-45d3-9c12-3e45249886a0)
